src/converters/UserGroupRelationshipConverter.ts

```diff
@@ -21,14 +21,22 @@ export function createUserGroupRelationships(users: User[]) {
     const parentKey = generateKey(GROUP_ENTITY_TYPE, user.group_id);
     const childKey = generateKey(USER_ENTITY_TYPE, user.id);
 
-    const relationship: UserGroupRelationship = {
+    const assignedRelationship: UserGroupRelationship = {
       _class: USER_GROUP_RELATIONSHIP_CLASS,
       _fromEntityKey: parentKey,
       _key: `${parentKey}_assigned_${childKey}`,
```

Comment:
I think this `_key` value is incorrect, since the docs say `onelogin_user` **ASSIGNED** `onelogin_group`. While not part of the specific task identified by #18, I think it would be good to address here since this code is being adjusted.

```suggestion
       _key: `${childKey}_assigned_${parentKey}`,
```